ironfish/src/rpc/routes/eth/sendTransaction.ts

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { LegacyTransaction } from '@ethereumjs/tx'
+import { Address } from '@ethereumjs/util'
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { GLOBAL_IF_ACCOUNT } from '../../../evm'
+import { FullNode } from '../../../node'
+import { legacyTransactionToEvmDescription } from '../../../primitives'
+import { EthUtils } from '../../../utils/eth'
+import { AssertSpending } from '../../../wallet/account/account'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+
+export type EthSendTransactionRequest = {
+  from: string
+  to?: string
+  gas?: string
+  gasPrice?: string
+  value?: string
+  data: string
```

Comment:
Does data have to be defined? I thought you could send nil for data

Reply:
made it optional